packages/local_auth/local_auth/CHANGELOG.md

```diff
@@ -1,7 +1,40 @@
-## NEXT
+## 2.0.0
 
+* Migrates plugin to federated architecture.
 * Adds OS version support information to README.
-* Switches over to default method implementation in new platform interface.
+* BREAKING CHANGE: Deprecated method `authenticateWithBiometrics` has been removed
+  Use `authenticate` instead.
+* BREAKING CHANGE: Parameters of `authenticate` have been changed.
+  
+  Example:
+```dart  
```

Comment:
Nit: I believe if you indent all of this code block, including the opening and closing backticks, two spaces (or maybe it's 4?) it will show as nested under the bullet point.